docs/network-flow-visibility.md

```diff
@@ -336,24 +338,34 @@ the Flow Aggregator adds the following fields to the flow records.
 
 #### IEs from Antrea IE Registry
 
-| IPFIX Information Element                 | Enterprise ID | Field ID | Type        |
-|-------------------------------------------|---------------|----------|-------------|
-| packetTotalCountFromSourceNode            | 56506         | 120      | unsigned64  |
-| octetTotalCountFromSourceNode             | 56506         | 121      | unsigned64  |
-| packetDeltaCountFromSourceNode            | 56506         | 122      | unsigned64  |
-| octetDeltaCountFromSourceNode             | 56506         | 123      | unsigned64  |
-| reversePacketTotalCountFromSourceNode     | 56506         | 124      | unsigned64  |
-| reverseOctetTotalCountFromSourceNode      | 56506         | 125      | unsigned64  |
-| reversePacketDeltaCountFromSourceNode     | 56506         | 126      | unsigned64  |
-| reverseOctetDeltaCountFromSourceNode      | 56506         | 127      | unsigned64  |
-| packetTotalCountFromDestinationNode       | 56506         | 128      | unsigned64  |
-| octetTotalCountFromDestinationNode        | 56506         | 129      | unsigned64  |
-| packetDeltaCountFromDestinationNode       | 56506         | 130      | unsigned64  |
-| octetDeltaCountFromDestinationNode        | 56506         | 131      | unsigned64  |
-| reversePacketTotalCountFromDestinationNode| 56506         | 132      | unsigned64  |
-| reverseOctetTotalCountFromDestinationNode | 56506         | 133      | unsigned64  |
-| reversePacketDeltaCountFromDestinationNode| 56506         | 134      | unsigned64  |
-| reverseOctetDeltaCountFromDestinationNode | 56506         | 135      | unsigned64  |
+| IPFIX Information Element                 | Field ID | Type        | Description |
+|-------------------------------------------|----------|-------------|-------------|
+| packetTotalCountFromSourceNode            | 120      | unsigned64  | The cumulative number of packets from source to destination for this flow since the flow starts, based on the records sent from the source node. |
+| octetTotalCountFromSourceNode             | 121      | unsigned64  | The cumulative number of octets from source to destination for this flow since the flow starts, based on the records sent from the source node. |
+| packetDeltaCountFromSourceNode            | 122      | unsigned64  | The number of packets from source to destination since the previous report for this flow at the observation point, based on the records sent from the source node. |
```

Comment:
I would simplify a bit the phrasing: "The number of packets for this flow as reported by the source Node, since the previous report for this flow at the observation point"

let me know if that works

Reply:
Thanks, Antonin. I simplified it accordingly.